cns/restserver/internalapi.go

```diff
@@ -94,32 +92,41 @@ func (service *HTTPRestService) SyncNodeStatus(dncEP, infraVnet, nodeID string,
 	}
 	service.RUnlock()
 
-	// check if the version is valid and save it to service state
-	for ncid, nc := range ncsToBeAdded {
-		nmaReq := nmagent.NCVersionRequest{
-			AuthToken:          nc.AuthorizationToken,
-			NetworkContainerID: nc.NetworkContainerid,
-			PrimaryAddress:     nc.PrimaryInterfaceIdentifier,
-		}
-
-		ncVersionURLs.Store(nc.NetworkContainerid, nmaReq)
-		waitingForUpdate, _, _ := service.isNCWaitingForUpdate(nc.Version, nc.NetworkContainerid)
-
-		body, _ = json.Marshal(nc)
-		req, _ = http.NewRequest(http.MethodPost, "", bytes.NewBuffer(body))
-		req.Header.Set(common.ContentType, common.JsonContent)
+	skipNCVersionCheck := false
+	ctx, cancel := context.WithTimeout(context.Background(), ContextTimeOut)
```

Comment:
This may already be an existing pattern so I won't block on this, but I would recommend renaming the `ContextTimeout` variable to something meaningful, like `nmaAPICallTimeout`

Reply:
I can update it with a nma specific varaible and rename it.